package.json

```diff
@@ -128,6 +128,7 @@
     "normalize.css": "^8.0.0",
     "react": "^16.5.2",
     "react-app-polyfill": "^0.1.3",
+    "react-popover": "^0.5.10",
```

Comment:
Hi @khus29 react-popover seems to be using react portals and with react portals one issue is handling keyboard tabbing. Please check if react-popover supports tabbing.